### DISCUSSION TITLE
Any way to import data from SimpleTabGroups

### DISCUSSION THREAD

**Discussion by ItsABlackScreen** [mbnuqw/sidebery#1406] [Q&A]
Hi, i was wondering if there was a way to import my existing group and tabs from SimpleTabGroups
https://github.com/Drive4ik/simple-tab-groups/

Doing it manually is not an option as i have about 1500 tabs.
Any help would be appreciated.

**Comment by ItsABlackScreen** [comment]
I ended up figuring it out on my own, for future reference. 
In STG, 
1. Open up a group, right click on a tab and select all. 
2. Save them to a bookmark folder with the group name.
3. Do this for all the groups and then delete them, and disable STG.
4. In sideberry create a pane and choose import from bookmarks. 

Leaving this open because maybe there is a better way to do this, i had a few groups so it wasn't so tedious, but someone with 999 groups with 999 tabs might have a problem.

**Comment by oktayacikalin** [comment]
Story:
I just migrated from STG to Sidebery. Over the last decade I accumulated nearly 2000 bookmarks (never read later, interesting stuff on already closed sites etc., and some real projects) resulting in about 50 groups in STG. Good point in time to leave old stuff behind.
So I created a new profile and installed every plugin I really need. Should I miss something, I can always open up my old profile and retrieve the missing data/plugin/whatever. Some configurations could be copied, floccus made it really easy, some made it difficult. I'm not using Firefox on mobile, so I did not set up account syncing. I'm syncing most of my stuff with my Nextcloud instance.

Action:
I had STG configured to manage a bookmark-tree of everything.
So I could use Firefox's bookmark export and import functionality to copy them over.
Whenever I need a group, I just restore it to a panel and continue working with it.

I don't know, if using the bookmarks-panel functionality would be a better choice, yet.
